gwlfe/WriteOutputFiles.py

```diff
@@ -7,9 +7,717 @@
 
 
 def WriteOutput(z):
-    print('WriteOutput')
+    # DIMENSION VARIABLES FOR PREDICT CALCULATION AND SCENARIO FILE
+    AvOtherLuSed = 0
+    AvOtherLuNitr = 0
+    AvOtherLuPhos = 0
+    TotSewerSys = 0
+    TotNormSys = 0
+    TotShortSys = 0
+    TotSeptSys = 0
+    TotAvLuErosion = 0
+    AvTotalSed = 0
+    AvDisN = 0
+    AvTotalN = 0
+    AvDisP = 0
+    AvTotalP = 0
+    n2t = 0
+    n6t = 0
+    n13t = 0
+    n24t = 0
+
+    z.SumTotArea = z.SumTotArea + z.TotArea
+
+    # INSERT VALUES FOR BMP SCENARIO FILE FOR PREDICT APPLICATION
+    for l in range(0, z.NLU):
+        z.AvLuSedYield[l] = (z.AvLuSedYield[l] * z.RetentFactorSed) * (1 - z.AttenTSS)
+        z.AvLuDisNitr[l] = (z.AvLuDisNitr[l] * z.RetentFactorN) * (1 - z.AttenN)
+        z.AvLuTotNitr[l] = (z.AvLuTotNitr[l] * z.RetentFactorN) * (1 - z.AttenN)
+        z.AvLuDisPhos[l] = (z.AvLuDisPhos[l] * z.RetentFactorP) * (1 - z.AttenP)
+        z.AvLuTotPhos[l] = (z.AvLuTotPhos[l] * z.RetentFactorP) * (1 - z.AttenP)
+
+    # SET THE SCENARIO VALUES TO LANDUSE LOADS
+    for l in range(0, z.NRur):
+        if (z.Landuse[l].lower() == "hay/past"):
```

Comment:
Because different parts of the GMS file have slightly different spellings of these land types I have created "enum" values here https://github.com/WikiWatershed/gwlf-e/blob/develop/gwlfe/enums.py#L64.

So instead of `if (z.Landuse[l].lower() == "hay/past")` this should be `if z.Landuse[l] is enums.LandUse.HAY_PAST`.